mopro-core/Cargo.toml

```diff
@@ -36,6 +39,7 @@ ark-groth16 = { version = "=0.4.0", default-features = false, features = [
 ] }
 ark-relations = { version = "0.4", default-features = false }
 ark-zkey = { path = "../ark-zkey" }
+witness = { git = "https://github.com/philsippl/circom-witness-rs.git"}
```

Comment:
`path = "../../witness` (e.g.)

and if making changes to it you want other people to use, can have comment to your forked git hash